linalg/simd/vsx.hpp

```diff
@@ -12,6 +12,10 @@
 #ifndef MFEM_SIMD_VSX_HPP
 #define MFEM_SIMD_VSX_HPP
 
+#if !(defined(__CUDACC__) || defined(__HIPCC__))
```

Comment:
This means you can never use these intrinsics with CUDA/HIP. When using `nvcc`, is there no host compiler with which these work?